lookout/style/typos/tests/test_generation.py

```diff
@@ -50,7 +50,6 @@ def test_save_load(self):
             print(buffer.tell())
             buffer.seek(0)
             generator2 = CandidatesGenerator().load(buffer)
-
```

Comment:
We still need to mark this test as `@unittest.expectedFailure`. I investigated the problem and it is not easily fixed by those changes in `lookout/style/typos/generation.py`. The core root is I think because sometimes it compares a numpy array to an ASDF proto-array which casts itself to a numpy array and is not an instance of ndarray. So this is the fault of the test after all.